fem/pbilinearform.hpp

```diff
@@ -150,6 +150,16 @@ class ParBilinearForm : public BilinearForm
        vectors on the true dofs. */
    void TrueAddMult(const Vector &x, Vector &y, const double a = 1.0) const;
 
+   using BilinearForm::InnerProduct;
```

Comment:
Do we need this here? `BilinearForm::InnerProduct(const Vector &, const Vector &)` should be accessible without problems, or am I missing something?

Reply:
True, it is not needed at the moment.